src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2799,6 +2815,31 @@ public Problem loadCCSProblemFiles(IInternalContest contest, String dataFileBase
         return problem;
     }
 
+    protected void loadDataFiles(Problem problem, ArrayList<SerializedFile> dataFiles, ArrayList<SerializedFile> answerFiles, String sampleDataDirectory, boolean loadExternalFile) {
+        
+        String[] inputFileNames = getFileNames(sampleDataDirectory, ".in");
+        String[] answerFileNames = getFileNames(sampleDataDirectory, ".ans");
+
```

Comment:
Shouldn't a check be made to make sure that inputFileNames.length == answerFileNames.length ?